### ISSUE TITLE
no line width with lineWidth: 0 or -1 ?

### ISSUE THREAD

**Issue by lts372005** [dprint/dprint#426]
I'm a vscode word wrap person. so I'd like dprint not making long line into multi line

The problem being
With vscode sidebar, available line width is 80 ish
Without vscode sider, available line width is 120 ish

lineWidth 80 is too wasted when I use vscode without sidebar,

lineWidth 120 causing me either have to scroll left and right (which is annoying) or turn on vscode word warp which result in instead of everything in one line and vscode word wrap make it 2 lines, dprint break the long line into 2 and then vscode continue breaking the longer line, resulting in 3

You might say: what about lineWidth 2 147 483 647
It works most of the time but then I have JavaScript and JSON objects that I want to keep multiline and dprint just make it one liner

TL;DR I USE WORD WARP AND I WANT TO DISABLE LINEWDITH

**Comment by joshuafcole** [comment]
Did you find a solution to this?
